awx/ui_next/src/components/Sparkline/HostStatusIcon.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { string } from 'prop-types';
+import {
+  ChangedBottom,
+  ChangedTop,
+  FailedBottom,
+  FailedTop,
+  FinishedJob,
+  SkippedBottom,
+  SkippedTop,
+  SuccessfulBottom,
+  SuccessfulTop,
+  UnreachableBottom,
+  UnreachableTop,
+} from './shared/StatusIcon';
+
+const HostStatusIcon = ({ status }) => {
+  return (
+    <div>
+      {status === 'changed' && (
+        <FinishedJob>
```

Comment:
this would be used as `<StatusIcon status={status} />` or something like that.  In terms of usage, we could delete HostStatusIcon and use StatusIcon itself.

FinishedJob vs. WaitingJob can be inferred from the status, it does not need to be passed.